src/components/Sidebar.vue

```diff
@@ -19,12 +19,7 @@
         </div>
       </div>
       <div class="flex mb-5 text-2xl justify-evenly">
-        <a class="mx-2 icon-btn" :title="t('button.toggle_dark')" @click="isDark = !isDark">
-          <carbon-moon v-if="isDark" />
-          <carbon-sun v-else />
-        </a>
-
-        <a class="mx-2 icon-btn" :title="t('button.toggle_langs')" @click="toggleLocales">
+        <a class="mx-2 icon-btn" :title="t('buttons.toggle_lang')" @click="toggleLocales">
```

Comment:
I don’t know why @antfu used a link. This should be a button, it does not redirect to a page, does it ?